db/schema.rb

```diff
@@ -218,6 +218,7 @@
     t.date "date_to"
     t.uuid "allocation_id"
     t.string "rejection_reason"
+    t.uuid "original_move_id"
```

Comment:
might need an index if being searched on (see earlier comment)

Reply:
Added a foreign key for this (should have added that before really)